impress-org/givewp-next-gen
# Feature: Amount Field settings updates

## Description



This PR adds the Price Option, Set Price, and Custom Amount settings to the Amount Field block.

## Visuals



![Peek 2023-01-24 11-40](https://user-images.githubusercontent.com/10858303/214386044-e17dda58-20c4-4b0d-8e00-c8c48ee8e2b0.gif)


## Testing Instructions



Switch between Price Options, update the Set Price, and toggle the Custom Amount settings. The amount field should update accordingly.
